policy/examples/tiered-parking-fees-per-hour.json

```diff
@@ -0,0 +1,57 @@
+{
+  "updated": 0,
+  "version": "1.2.0",
+  "data": {
+    "policy": [
+      {
+        "name": "Tiered Dwell Time Example",
+        "description": "First hour $2, second hour $4, every hour onwards $10",
+        "policy_id": "2800cd0a-7827-4110-9713-b9e5bf29e9a1",
+        "start_date": 1558389669540,
+        "publish_date": 1558389669540,
+        "end_date": null,
+        "prev_policies": null,
+        "provider_ids": [],
+        "currency": "USD",
+        "rules": [
+          {
+            "name": "> 2 hours",
+            "rule_id": "9cd1768c-ab9e-484c-93f8-72a7078aa7b9",
+            "rule_type": "time",
+            "rule_units": "hours",
+            "geographies": ["0c77c813-bece-4e8a-84fd-f99af777d198"],
+            "statuses": { "available": [], "non_operational": [] },
+            "vehicle_types": ["bicycle", "scooter"],
+            "maximum": 2,
```

Comment:
Maximum in this case would be "If a vehicle surpasses this maximum, it's matched by this rule". Similarly, a minimum would be "If a vehicle is _under_ this minimum, it's matched by this rule". I can't think of any good real `minimum` time policies off the top of my head, but essentially you _could_ encode something like "Scooters must be parked here in state x for a minimum of 1 hour prior to doing anything else". Maybe this could be used to encode some rules around off-hours usage perhaps?

You can think of the matching along the lines of:
```
if (time_since_last_event > maximum) {
  // maybe a match, check other criteria (geos, states, etc...)
} else if (time_since_last_event < minimum) {
  // maybe a match, check other criteria (geos, states, etc...)
} else {
  definitely not a match
}
```